packages/c/justfile

```diff
@@ -51,10 +51,12 @@ strict-docker +ARGS='':
 memcheck +ARGS='': build-dev
   #!/usr/bin/env bash
   file="valgrind-sshnpd-$(date +%s).log"
+  # child-silient-after-fork=yes: ignore session forks, we don't care
   valgrind --log-file="$file" \
   --tool=memcheck \
   --leak-check=full \
   --show-leak-kinds=all \
+  --child-silent-after-fork=yes \
```

Comment:
When we leave a session open, this hides the leaks which we expect to occur: https://github.com/Mbed-TLS/mbedtls/issues/9958